test/rules/android_sdk_repository/android_sdk_repository_test.sh

```diff
@@ -28,10 +28,202 @@ source "${RUNFILES_DIR:-/dev/null}/$f" 2>/dev/null || \
 source "$(rlocation rules_android/test/bashunit/unittest.bash)" || \
   (echo >&2 "Failed to locate bashunit.sh" && exit 1)
 
-ENABLE_PLATFORMS=false
+source "$(rlocation rules_android/test/rules/android_sdk_repository/android_helper.sh)" || \
+  (echo >&2 "Failed to locate android_helper.sh" && exit 1)
 
-# Source and run the actual tests from test_lib.sh
-source "$(rlocation rules_android/test/rules/android_sdk_repository/test_lib.sh)" || \
-  (echo >&2 "Failed to locate test_lib.sh" && exit 1)
+# Tests for Android Sdk Repository
+
+# Test that the dummy SDK exists.
+function test_dummy_sdk() {
+  # Create android SDK
+  local sdk_path="$(create_android_sdk_basic)"
+
+  cat >> WORKSPACE <<EOF
```

Comment:
Is this loading the native android_sdk_repository into a WORKSPACE file? Ideally we'd want to load the Starlark one.

Reply:
See android_helper.sh, line 84: this is testing the Starlark version.